Headlines/src/Controllers/TrendingCardsViewController.swift

```diff
@@ -62,6 +62,18 @@ class TrendingCardsViewController: UICollectionViewController {
         })
     }
     
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        guard let flowLayout = collectionView?.collectionViewLayout as? UICollectionViewFlowLayout,
+            let collectionViewSize = self.collectionView?.bounds.size else {
+                return
+        }
+        if UIDevice.current.userInterfaceIdiom == .pad {
+            let columns: CGFloat = UIApplication.shared.statusBarOrientation.isLandscape ? 3.0 : 2.0
+            flowLayout.itemSize = CGSize(width: ((collectionViewSize.width - 20 - (columns-1)*10) / columns), height: 235)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/f609a77816dcd1d12fd0f37d58964ee47f86c075/Headlines/src/Controllers/TrendingCardsViewController.swift#L73"></span>Line should be 120 characters or less: currently 122 characters</td>
    </tr>
  </tbody>
</table>